ibis/backends/dask/core.py

```diff
@@ -185,3 +484,10 @@ def execute_and_reset(
     elif isinstance(result, dd.Series):
         return result.reset_index(drop=True)
     return result
+
+
+@compute_time_context.register(ops.Node)
+def compute_time_context_default(
+    node, timecontext: Optional[TimeContext] = None, **kwargs
+):
+    return [timecontext for arg in node.inputs if is_computable_input(arg)]
```

Comment:
I _think_ you can also drop this redefinition - the dask backend doesn't really use (or modify) `compute_time_context` currently since we have no windowing, so just using the pandas one should be fine for now.